modules/publication/jsx/projectFields.js

```diff
@@ -403,11 +411,15 @@ class ProjectFormFields extends React.Component {
           items={this.props.formData.voiFields}
           btnLabel="Add Variable of Interest"
         />
-        <StaticElement
-          text={voiHelp}
-        />
+        <div className="row form-group">
+          <label className="col-sm-3 control-label">
+          </label>
+          <div className="col-sm-9">
+            <div className="form-control-static">{voiHelp}</div>
+          </div>
+        </div>
```

Comment:
what's the reason for this change? it looks to me that the html here is the same as is in the StaticElement render function

Reply:
@zaliqarosli There is a slight change if you compare the render of StaticElement to my change. Turns out this change solved the console error: "Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>." but maybe I should modify the StaticElement and get rid of the use of <p>. What do you think?

Reply:
could you try changing to <div> tags in `voiHelp` to <span>? I think this would be the best solution

Reply:
@zaliqarosli sure why not